sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/models/CallMediaRecognizeDtmfOptions.java

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.communication.callautomation.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+/** The Recognize configurations specific for Dtmf. **/
+public class CallMediaRecognizeDtmfOptions extends CallMediaRecognizeOptions {
+
+    /*
+     * Time to wait between DTMF inputs to stop recognizing.
+     */
+    @JsonProperty(value = "interToneTimeoutInSeconds")
+    private Integer interToneTimeoutInSeconds;
```

Comment:
We are dropping the `InSeconds` suffix.
In C# we changed this to be a `TimeSpan` object, we need to find the equivalent in Java. I think it is `OffsetTime`? You can check on the rest of the SDK which is it.

Reply:
You'll need to translate from that object to seconds when creating the request, though.